lib/concurrent-ruby/concurrent/utility/processor_counter.rb

```diff
@@ -41,6 +42,10 @@ def cpu_quota
         @cpu_quota.value
       end
 
+      def cpu_requests
+        @cpu_requests.value / 1024
```

Comment:
This will NoMethodError if there is neither cgroup 1 or 2.
I think it makes sense to move the `/ 1024` in `compute_cpu_requests`